<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbl_seed">
       <property name="font">
        <font>
         <family>Arial</family>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Случайно добавить задания</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="lndt_seed">
         <property name="font">
          <font>
           <family>Arial</family>
           <bold>false</bold>
          </font>
         </property>
         <property name="placeholderText">
          <string>Ключ генерации (оставьте пустым для случайного)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pshbttn_addseed">
         <property name="minimumSize">
          <size>
           <width>112</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>112</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="lbl_types">
       <property name="font">
        <font>
         <family>Arial</family>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Случайно добавить задание(-я) определённой линии</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QComboBox" name="cmbbx_types">
         <property name="font">
          <font>
           <family>Arial</family>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spnbx_types">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pshbttn_addtypes">
         <property name="minimumSize">
          <size>
           <width>112</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>112</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="lbl_id">
       <property name="font">
        <font>
         <family>Arial</family>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Вручную добавить задание</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLineEdit" name="lndt_id">
         <property name="font">
          <font>
           <family>Arial</family>
           <bold>false</bold>
          </font>
         </property>
         <property name="placeholderText">
          <string>Идентификатор задания</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pshbttn_addid">
         <property name="minimumSize">
          <size>
           <width>112</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>112</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTableView" name="tablvw_questions">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <bold>false</bold>
        </font>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragDropMode::NoDragDrop</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::DropAction::IgnoreAction</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="pshbttn_clear">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Очистить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pshbttn_delete">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pshbttn_compile">
     <property name="font">
      <font>
       <family>Arial</family>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Составить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
